build/nuget-release.yml

```diff
@@ -85,38 +85,14 @@ stages:
             inputs:
               artifact: 'Templates'
               path: '$(Build.SourcesDirectory)/templates'
-          - task: GitHubRelease@0
-            displayName: 'Create GitHub Release (Conditional Step)'
-            inputs:
-              gitHubConnection: 'GitHub (arcus-automation - OAuth)'
-              repositoryName: 'arcus-azure/arcus.templates'
-              tagSource: manual
-              tag: 'v$(Build.BuildNumber)'
-              title: 'v$(Build.BuildNumber)'
-              releaseNotesSource: input
-              releaseNotes: |
-                Install new version via [NuGet](https://www.nuget.org/packages/Arcus.Templates.WebApi/$(Build.BuildNumber))
-                ```shell
-                > dotnet new --install Arcus.Templates.WebApi::$(Build.BuildNumber)
-                ```
-            condition: not(contains(variables['Build.BuildNumber'], '-'))
-          - task: GitHubRelease@0
-            displayName: 'Create GitHub Pre-Release (Conditional Step)'
-            inputs:
-              gitHubConnection: 'GitHub (arcus-automation - OAuth)'
+          - template: github/create-release.yml@templates
+            parameters:
               repositoryName: 'arcus-azure/arcus.templates'
-              tagSource: manual
-              tag: 'v$(Build.BuildNumber)'
-              title: 'v$(Build.BuildNumber)'
-              releaseNotesSource: input
               releaseNotes: |
                 Install new version via [NuGet](https://www.nuget.org/packages/Arcus.Templates.WebApi/$(Build.BuildNumber))
                 ```shell
-                > dotnet new --install Arcus.Templates.WebApi::$(Build.BuildNumber)
+                > dotnet new --install Arcus.Templates.WebApi --Version $(Build.BuildNumber)
```

Comment:
Is this correct? AFAIK it's not

Reply:
Yeah, not sure about this. All the other NuGet release YAML build files uses `-Version` with the build number. So... Not sure why this would be any different?

Reply:
Because it's a dotnet template

Reply:
🤦‍♂

Reply:
Thanks.

Reply:
No worries, that's what reviews are for